packages/gamut/src/Badge/index.tsx

```diff
@@ -1,18 +1,55 @@
+import { StyleProps } from '@codecademy/variance';
+import styled from '@emotion/styled';
 import React from 'react';
 
+import { variant } from '../../../gamut-styles/dist';
```

Comment:
Auto import doing weird stuff

```suggestion
import { variant } from '@codecademy/gamut-styles';
```